### DISCUSSION TITLE
Curiously, what do the word "red" and  "carrot" mean?

### DISCUSSION THREAD

**Discussion by caozx1110** [larics/icuas22_competition#21] [General]
"red" may refer to the UAV.

And what about the carrot ?

lol.

**Comment by lmark1** [comment]
Hello @caozx1110,

Red is the UAV namespace.

The inspiration for the name carrot (a.k.a. the current UAV referent trajectory point) comes from the _carrot-on-a-stick_ concept. See picture below - the donkey represents the UAV.
![image info](https://peopletalkonline.ca/wp-content/uploads/2015/04/carrot-stick.jpg)

Hope that clears it up. :)

**Reply by caozx1110** [reply]
OK, I get it !
Thanks a lot!

**Comment by fpetric** [comment]
And "red" actually means the color red, it is a legacy of color coding our UAVs :). Here are our "red" and "yellow" UAVs, see the 3D printed parts :)
![image](https://user-images.githubusercontent.com/5979567/156344618-f8dc1e30-925c-4fff-a384-c05b28ed1086.png) ![image](https://user-images.githubusercontent.com/5979567/156344639-890fc0bd-c9ab-4782-b258-6a2227fabf44.png)
